server/src/domain/album/album.service.spec.ts

```diff
@@ -153,44 +155,71 @@ describe(AlbumService.name, () => {
   describe('create', () => {
     it('creates album', async () => {
       albumMock.create.mockResolvedValue(albumStub.empty);
-
-      await expect(sut.create(authStub.admin, { albumName: 'Empty album' })).resolves.toEqual({
+      userMock.get.mockResolvedValue(userStub.user1);
+      const expectedResult = {
         albumName: 'Empty album',
         description: '',
         albumThumbnailAssetId: null,
         assetCount: 0,
-        assets: [],
-        createdAt: expect.anything(),
+        assets: [
+          {
+            id: '123',
+            checksum: '',
+            deviceAssetId: '',
+            deviceId: '',
+            duration: '',
+            fileCreatedAt: new Date('1970-01-01'),
+            fileModifiedAt: new Date('1970-01-01'),
+            isArchived: false,
+            isFavorite: false,
+            originalFileName: '',
+            originalPath: '',
+            ownerId: '',
+            resized: false,
+            thumbhash: '',
+            type: AssetType.IMAGE,
+            updatedAt: new Date('1970-01-01'),
+          },
+        ],
+        createdAt: new Date('1970-01-01'),
         id: 'album-1',
-        owner: {
-          email: 'admin@test.com',
-          firstName: 'admin_first_name',
-          id: 'admin_id',
-          isAdmin: true,
-          lastName: 'admin_last_name',
-          oauthId: '',
-          profileImagePath: '',
-          shouldChangePassword: false,
-          storageLabel: 'admin',
-          createdAt: new Date('2021-01-01'),
-          deletedAt: null,
-          updatedAt: new Date('2021-01-01'),
-          externalPath: null,
-          memoriesEnabled: true,
-        },
+        owner: userStub.admin,
         ownerId: 'admin_id',
         shared: false,
-        sharedUsers: [],
+        sharedUsers: [userStub.user1],
         startDate: undefined,
         endDate: undefined,
         hasSharedLink: false,
-        updatedAt: expect.anything(),
-      });
+        updatedAt: new Date('1970-01-01'),
+      };
+      const mapAlbumWithAssetsSpy = jest.spyOn(albumResponse, 'mapAlbumWithAssets').mockReturnValue(expectedResult);
```

Comment:
I mean I could come up with an excuse but actually I was just too lazy to think about what the mapping would return and since the mappings had all been tested already I didn't see a disadvantage with just mocking `mapAlbumWithAssets`.

Reply:
In that case, just do await sut.method() without an expect and delete the mock, spy, and expected response altogether.

Reply:
Depending on who's gonna implement the changes, I can do whatever you prefer or you just implement it as you like. I'm perfectly fine with both

Reply:
I am fine with either (1) validate the actual returned data correctly goes through the mapper and results in the right structure or (2) ignore the returned data entirely and delete the expect, spy, mock, expected data.

Personally, I am also lazy and would probably choose (2) myself :laughing:

Reply:
I've just looked at the coverage and `mapAlbumWithAssets` is called 22x by unit tests alone and every path is covered multiple times inside `mapAlbum`. So (obviously without laziness in mind xD), we can safely choose (2).